pkg/harvester/l10n/zh-hans.yaml

```diff
@@ -286,8 +296,12 @@ harvester:
       run: 运行策略
       configure: 配置
       mode: 模式
-      modeLink: 模式 <a href="https://docs.harvesterhci.io/v1.0/host/host/#enable-ksmtuned" target="_blank"><i class="icon icon-info" /></a>
+      modeLink: 模式 <a href="https://docs.harvesterhci.io/next/host/#ksmtuned-mode" target="_blank"><i class="icon icon-info" /></a>
```

Comment:
use v1.1.0 release docs link

Reply:
Already updated the link in both the English and Chinese files. The v1.1 doc links should be available once https://github.com/harvester/docs/pull/206 is merged.